styles/style.css

```diff
@@ -4,7 +4,95 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
+.first-div {
+  background-color: #3b6890;
+}
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  text-align: center;
+  margin: auto;
+}
```

Comment:
When you finish the code, review and delete all unnesessary lines.
For example line 12 and 14 -> do you need them both?